server/routes/health.js

```diff
@@ -1,26 +1,10 @@
 const express = require('express');
-const { path } = require('ramda');
 
-const createHealthRouter = ({ healthService }) => {
+const createHealthRouter = () => {
   const router = express.Router();
 
   router.get('/', (_, res) => res.json({ status: 'OK' }));
-
-  router.get('/readiness', async (req, res, next) => {
-    try {
-      const healthStatus = await healthService.status();
-      const buildInfo = path(['app', 'locals', 'config', 'buildInfo'], req);
-      res.set('Content-Language', 'en-GB');
-      res.json({
-        buildNumber: path(['buildNumber'], buildInfo),
-        gitRef: path(['gitRef'], buildInfo),
-        gitDate: path(['gitDate'], buildInfo),
-        ...healthStatus,
-      });
-    } catch (exp) {
-      next(exp);
-    }
-  });
+  router.get('/readiness', (_, res) => res.json({ status: 'OK' }));
```

Comment:
The build info should be in a separate endpoint if it is still required. It is nothing to do with the 'health' of the app.